README.adoc

```diff
@@ -0,0 +1,143 @@
+
+- Openshift 3.9 cluster
+- Istio 0.7.1 with authentication installed on the aforementioned cluster.
+To install Istio simply follow one of the following docs:
+    * https://istio.io/docs/setup/kubernetes/quick-start.html
+    * https://istio.io/docs/setup/kubernetes/ansible-install.html
+- Enable automatic sidecar injection for Istio
+  * See https://istio.io/docs/setup/kubernetes/sidecar-injection.html[this] for details
+- Login to the cluster with the admin user
+
+NOTE: The `istiooc cluster up --istio=true` command from https://github.com/openshift-istio/origin/releases/[this project] perfectly satisfy the above requirements.
+
+== Environment preparation
+
+```bash
+    oc new-project circuit-breaker-mission
+```
+*CAUTION*:
+In order for Istio automatic sidecar injection to work properly the following Istio configuration needs to be in place:
+
+. The `policy` field is set to `disabled` in the `istio-inject` configmap  of the `istio-system` namespace
+. The `istio-sidecar-injector` `MutatingWebhookConfiguration` should not limit the injection to properly labeled namespaces
+. Expose services and Istio ingress
++
+```
+oc expose svc istio-ingress -n istio-system
+```
+The aforementioned configuration is not needed when the cluster has been setup using `istiooc`
+
+== Build and deploy the application
+
+=== With Fabric8 Maven Plugin (FMP)
+Execute the following command to build the project and deploy it to OpenShift:
+```bash
+mvn clean package fabric8:deploy -Popenshift
+```
+Configuration for FMP may be found both in pom.xml and `src/main/fabric8` files/folders.
+
+This configuration is used to define service names and deployments that control how pods are labeled/versioned on the OpenShift cluster. Labels and versions are key concepts for creating load-balanced or multi-versioned pods in a service.
+
+
+=== With Source to Image build (S2I)
+Run the following commands to apply and execute the OpenShift templates that will configure and deploy the applications:
+```bash
+find . | grep openshiftio | grep application | xargs -n 1 oc apply -f
+
+oc new-app --template=spring-boot-circuit-breaker-greeting -p SOURCE_REPOSITORY_URL=https://github.com/snowdrop/spring-boot-circuit-breaker-booster  -p SOURCE_REPOSITORY_REF=with-sse -p SOURCE_REPOSITORY_DIR=name-service
+oc new-app --template=spring-boot-circuit-breaker-name -p SOURCE_REPOSITORY_URL=https://github.com/snowdrop/spring-boot-circuit-breaker-booster  -p SOURCE_REPOSITORY_REF=with-sse -p SOURCE_REPOSITORY_DIR=name-service
+```
+
+= Use Cases
+== Access the application
+
+Run the following command to determine the appropriate URL to access our demo. Make sure you access the url with the HTTP scheme. HTTPS is NOT enabled by default:
+
+```bash
+oc get route spring-boot-circuit-breaker-greeting -o jsonpath='http://{.spec.host}{"\n"}'
+```
+
+The result of the above command is the istio-system istio-ingress URL. Open this URL in your a web browser.
+
+== Circuit breaker without Istio
```

Comment:
Why did you change the text here?

Reply:
Looks like it was brought in line with the other boosters?